### PR TITLE
Report issue

### LINKED ISSUE #68: Report Issue Functionality
- [x] Add Report Issue Modal
- [x] Connect Modal to Formspree API

### PR DESCRIPTION
**Pull request by milesba4**
Created Report Issue Modal

### CODE REVIEW

**Comment by LifeBringer** [comment]
Why all the conflicts?

**Comment by milesba4** [comment]
> Why all the conflicts?

Fixed the conflicts, I just added changes to a lot of files

**Review by gabremo — APPROVED** [review]
Amazing job, Miles!!!

File: capstone-project/ui/src/components/ProfilePage/ProfilePage.jsx [hunk 1]
```diff
@@ -18,6 +19,12 @@ export default function ProfilePage() {
   // if user selects studying in preference modal it will display study preference form
   const [isStudying, setIsStudying] = React.useState(false);
   const [isWorking, setIsWorking] = React.useState(false)
+
+
+  const {reportModal,setReportModal,toggleReportModal}= useAuthContext();
+
+  console.log("t:", reportModal)
```
Comment (Rebecca-CYLi) [inline capstone-project/ui/src/components/ProfilePage/ProfilePage.jsx]: console log?

**Review by Rebecca-CYLi — CHANGES_REQUESTED** [review]
1 console to remove and you good

File: capstone-project/ui/src/components/ProfilePage/ProfilePage.jsx (hunk 1 above)
Comment (gabremo) [reply capstone-project/ui/src/components/ProfilePage/ProfilePage.jsx]: Addressed!

**Review by Rebecca-CYLi — APPROVED** [review]
revision made, G2G

**Pull request merged**